src/inverseRobot.js

```diff
@@ -19,12 +19,19 @@
  * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
  *
  *
- * @param {object} robot
- *
- * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const property in robot) {
+    if (!inversedRobot.hasOwnProperty([robot[property]])) {
+      inversedRobot[robot[property]] = property;
+    } else {
+      return null;
+    }
```

Comment:
invert your if statement, that way you won't need `else`, try to avoid eles
```suggestion
    if (inversedRobot.hasOwnProperty([robot[property]])) {
      return null;
    } 
    
    inversedRobot[robot[property]] = property;
```